Model/ResourceModel/Log.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Bolt magento2 plugin
```

Comment:
Please change the file name and variable name too. Everywhere it is still called log or log factory